src/Components/App/App.tsx

```diff
@@ -1,94 +1,110 @@
-import React, { useState, FC } from 'react';
-import './App.css';
-import Header from '../Header/Header'
-import Form from '../Form/Form'
-import HomeCityCards from '../HomeCityCards/HomeCityCards'
-import { fetchCityForecast, fetchForecastDetails } from '../../apiCalls'
-import { WeatherLocation } from "../../model/Weather";
-import { Forecast } from '../../model/Weather'
-import { Route } from 'react-router-dom';
-import TodaysForecastDetails from '../TodaysForecastDetails/TodaysForecastDetails';
-import FiveDayForecastCardContainer from '../FiveDayForecastCardContainer/FiveDayForecastCardContainer';
+import { useState, FC, useEffect } from "react";
+import "./App.css";
+import Header from "../Header/Header";
+import Form from "../Form/Form";
+import HomeCityCards from "../HomeCityCards/HomeCityCards";
+import { fetchCityForecast, fetchForecastDetails } from "../../apiCalls";
+import { WeatherLocation, Coordinates, Forecast } from "../../model/Weather";
+import { Route } from "react-router-dom";
+import TodaysForecastDetails from "../TodaysForecastDetails/TodaysForecastDetails";
+import FiveDayForecastCardContainer from "../FiveDayForecastCardContainer/FiveDayForecastCardContainer";
 
 const App: FC = () => {
   const [cities, setCities] = useState<WeatherLocation[]>([]);
-  const [error, setError] = useState('');
-  const [warning, setWarning] = useState('');
+  const [error, setError] = useState("");
+  const [warning, setWarning] = useState("");
   const [currentCity, setCurrentCity] = useState<WeatherLocation | null>(null);
   const [forecastDetails, setForecastDetails] = useState<Forecast | null>(null);
 
   const resetAlerts = () => {
-    setError('');
-    setWarning('');
-  }
+    setError("");
+    setWarning("");
+  };
 
   let addCity = async (cityName: string) => {
     resetAlerts();
     const fetchedCity = await fetchCityForecast(cityName);
 
     if (!fetchedCity) {
       setError(`No location found called '${cityName}'`);
-    } else if (cities.find(city => city.id === fetchedCity.id)) {
+    } else if (cities.find((city) => city.id === fetchedCity.id)) {
       setWarning(`Location '${cityName}' is already in the list.`);
     } else {
       setCities([fetchedCity, ...cities]);
     }
```

Comment:
This is a great solution for now! Thanks Darla. I am not a robot. Beep beep boop.